content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,90 @@
+## Installation
+
+- **Added** a command line [Istio operator](docs/setup/install/operator/) to install and manage the Istio control plane.
+
+## Traffic Management
+
+- **Added** automatic determination of protocol types. This removes the need for you to explicitly classify ports as `http` or `tcp`.
```

Comment:
s/classify/name service/ as we always use `named service ports` in our docs in the past:  https://istio.io/docs/setup/kubernetes/additional-setup/requirements/

Reply:
accepted.